macOS/debugger.cpp

```diff
@@ -1574,6 +1759,7 @@ void Debugger::Init(int argc, char **argv) {
   if (option) strncpy(target_method, option, PATH_MAX);
 
   loop_mode = GetBinaryOption("-loop", argc, argv, loop_mode);
+  disable_aslr = GetBinaryOption("-disable_aslr", argc, argv, disable_aslr);
```

Comment:
4th argument in GetBinaryOption is the default value. Since disable_aslr is never initialized before this point, if "-disable_aslr" command line is not set, disable_aslr will be unpredictable. What we want here is probably
```
disable_aslr = GetBinaryOption("-disable_aslr", argc, argv, false);
```
(setting the default to false to keep the default in line with the existing behavior)

Reply:
done